src/CheckoutSdk/Sources/ISourcesClient.cs

```diff
@@ -0,0 +1,17 @@
+using System.Threading.Tasks;
+
+namespace Checkout.Sources
+{
+    /// <summary>
+    /// Defines the operations available on the Checkout.com Sources API.
+    /// </summary>
+    public interface ISourcesClient
+    {
+        /// <summary>
+        /// Request a source.
```

Comment:
Correct me if I'm wrong, but I understand you're implementing [this method](https://api-reference.checkout.com/#tag/Sources/paths/~1sources/post). If this is the case, what do you think about making the descriptions match?